src/shared/components/workbasket-filters/workbasket-filters.component.html

```diff
@@ -39,6 +39,6 @@ <h2 class="heading-h2" aria-label="Filters">Filters</h2>
     </div>
   </ng-container>
 
-  <button type="button" class="button" (click)="apply(true)" [disabled]="isApplyButtonDisabled()" >Apply</button> &nbsp;&nbsp;&nbsp;
-  <button type="button" (click)="reset()" class="button button-secondary">Reset</button>
+  <button type="button" class="button" (click)="apply(true)" [disabled]="isApplyButtonDisabled()" title="Apply filter" aria-label="Apply filter">Apply</button> &nbsp;&nbsp;&nbsp;
```

Comment:
_Still being annoying:_
So not a claustrophobic `aria-label` then. Unless it got some treatment or you're maliciously playing on its fears.